bigtable/examples/run_examples_utils.sh

```diff
@@ -156,7 +157,6 @@ function run_all_data_examples {
 
   # Create an instance to run these examples.
   $admin ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}"
-  trap '$admin ../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}" "${zone_id}"' EXIT
```

Comment:
Why the removal of all `trap`s?

Reply:
Because they remove the more important trap that kills the emulators.  Traps do not stack :disappointed:

Reply:
Would it be possible to have a single function that does all global cleanups and use that? Or just create a stack of them?

E.g., see:

* https://stackoverflow.com/questions/16115144/bash-save-and-restore-trap-state-easy-way-to-manage-multiple-handlers-for-trap
* https://stackoverflow.com/a/7287873